subprojects/gradle/runner/service/src/main/kotlin/com/avito/runner/service/model/TestCase.kt

```diff
@@ -7,7 +7,7 @@ data class TestCase(
     val methodName: String,
     val deviceName: String
 ) : Serializable {
-    val testName: String = "$className.$methodName"
+    val testName: String = "$className.$methodName.$deviceName"
```

Comment:
I check all places where we use that property
Now we will differentiate tests by API

Reply:
It seems that the value is used for debugging.
Maybe we should express it explicitly in API.
`testName` looks like a stricter contract.